integration-tests/utils/integration-helpers.js

```diff
@@ -192,11 +192,19 @@ const enableDataCatalog = async (page, testUrl, token) => {
   await dismissNotifications(page)
 }
 
+const goToLinkWorkspacePage = async (page) => {
```

Comment:
I was imagining you would keep this within one of the tests that uses it (I don't think it's general enough to go in integration-helpers). And you can put `await enableDataCatalog(page, testUrl, token)` in it also.

Reply:
Oh I see! I find keeping `enableDataCatalog` separate makes the test read better but I'm fine either way

Reply:
Maybe call it something like "linkDataToWorkspace"? Then it would make sense that it includes enabling the data catalog (which it is interacting with in the test anyway).